generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management.component.ts.ejs

```diff
@@ -103,16 +105,32 @@ export class <%= entityAngularName %>Component implements OnInit, OnDestroy {
         <%_ } _%>
         <%_ if (pagination === 'pagination') { _%>
 
-    protected paginate<%= entityClassPlural %>(data: I<%= entityAngularName %>[] | null, headers: HttpHeaders): void {
+    protected onSuccess(data: I<%= entityAngularName %>[] | null, headers: HttpHeaders, page: number): void {
         <%_ if (databaseType !== 'cassandra') { _%>
         this.totalItems = Number(headers.get('X-Total-Count'));
+        this.page = page;
+        <%_ if (searchEngine === 'elasticsearch') { _%>
+        this.ngbPaginationPage = this.page;
         <%_ } _%>
-        if (data) {
-            this.<%= entityInstancePlural %> = data;
-        } else {
-            this.<%= entityInstancePlural %> = [];
-        }
+        this.router.navigate(['/<%= entityUrl %>'], {
+          queryParams: {
```

Comment:
minor comment: you switch here to 2 spaces instead of 4 ?

Reply:
Good catch! I corrected indentation. This was caused by having different spacing in generator templates and in final generated app formatted by Prettier:
* in generator templates is used 4 space indentation
* Prettier formats indentation to 2 spaces

And in this place I missed fixing indentation after copying from tested sample application.

I also rebased with latest master to pass CI.